pyqtorch/circuit.py

```diff
@@ -207,3 +207,25 @@ def dagger(self, values: dict[str, Tensor]) -> Tensor:
 
     def jacobian(self, values: dict[str, Tensor]) -> Tensor:
         return values[self.param_name] * ones_like(self.unitary(values))
+
+
+def hea(n_qubits: int, n_layers: int, param_name: str) -> tuple[ModuleList, ParameterDict]:
```

Comment:
```suggestion
def hea(n_qubits: int, depth: int, param_name: str) -> tuple[ModuleList, ParameterDict]:
```
Minor thing but just to be consistent with the current qadence